botorch/test_functions/synthetic.py

```diff
@@ -261,6 +264,7 @@ def __init__(
         noise_std: float | None = None,
         negate: bool = False,
         bounds: list[tuple[float, float]] | None = None,
+        dtype: torch.dtype = torch.double,
```

Comment:
missing dtype arg in docstring below

Reply:
Added the docstring, hence resolving this conversation.